tools/testbench/topology.c

```diff
@@ -614,15 +617,22 @@ int load_process(void *dev, int comp_id, int pipeline_id,
 	struct sof_ipc_comp_process process = {0};
 	struct sof_ipc_comp_process *process_ipc;
 	struct snd_soc_tplg_ctl_hdr *ctl = NULL;
+	struct sof_ipc_comp_ext comp_ext;
 	char *priv_data = NULL;
 	int size;
 	int ret = 0;
+	int i;
 
 	size = widget->priv.size;
-	ret = tplg_load_process(comp_id, pipeline_id, size, &process, file);
+	ret = tplg_load_process(comp_id, pipeline_id, size, &process, file, &comp_ext);
 	if (ret < 0)
 		return ret;
 
+	printf("debug uuid:\n");
+	for (i = 0; i < SOF_UUID_SIZE; i++)
+		printf("%u ", comp_ext.uuid[i]);
+	printf("\n");
+
```

Comment:
You need to remove the next "if (process.comp.type == SOF_COMP_NONE)" test to pass with non-legacy component.

Reply:
yes, sorry was testing only with modified iir, will change.